package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "docs",
+  "name": "keep",
```

Comment:
still some strange change here. why do you change the location of the file?

Reply:
unintentionally, it was done